test-case/manual/jack_detection_boot.md

```diff
@@ -0,0 +1,60 @@
+# Jack detection for headset/HDMI/DP during boot test
+Jack detection for headset/HDMI/DP during boot is part of a series of manual
+tests.
+
+# Preconditions
+1. System is powered off.
+
+# Test Description
+* Plug headset into unit, and determine if status is on as expected after power
+on and off after power down and removal.
+* Repeat for both HDMI and DisplayPort.
+
+## Headset
+1. Plug headset into 3 1/2 mm jack.
+2. Power on system.
+3. Check amixer contents via:
+    ```
+    amixer contents
```

Comment:
```suggestion
    amixer contents | grep -i jack -A 3
```

Reply:
I've been playing with ways to make this easier on the user (me). Its fairly annoying to have to rerun the script every time I plug / unplug & I was looking for a way to have the plug event update as it happens, without me re running commands.

This seems to do exactly what I want. What do you think @xiulipan ?

`watch -d -n 1 "amixer contents | grep -i jack -A 2"`

Reply:
Re-reading this, I'd like to clarify, the `watch -d ....` really only applies to the other two jack detection tests, not really the (this) boot test.

Reply:
@stripes416 You can come with advanced CMD to save time.
They will to be base for the automation test later.